src/main/java/com/validationtask/task2/ProductAPIExceptionHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.validationtask.task2;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestControllerAdvice
+public class ProductAPIExceptionHandler {
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<ErrorResponse> handleMethodArgumentNotValidException(MethodArgumentNotValidException e) {
```

Comment:
クラスメイとメソッドは 1 行あけるのが better です。
```suggestion
public class ProductAPIExceptionHandler {

    @ExceptionHandler(MethodArgumentNotValidException.class)
    public ResponseEntity<ErrorResponse> handleMethodArgumentNotValidException(MethodArgumentNotValidException e) {
```

Reply:
@yoshi-koyama 
クラスメイとメソッドの間に 1 行あけましたので確認お願いします。
https://github.com/linleck/validation-tutorial-linleck/pull/2/commits/bca8199e74b448a1150cae42795b8c6233c32220

Reply:
コミットハッシュ間違ってませんか？

Reply:
いや。合ってますね。
大丈夫です👌